<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PorosityAnalyser</class>
 <widget class="QMainWindow" name="PorosityAnalyser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,0">
    <item>
     <widget class="QWidget" name="mainArea" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="selectionsExplorer" native="true">
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="selectionsHeader" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>14</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="tbTreeView">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string extracomment="Selections">Selections</string>
         </property>
         <property name="statusTip">
          <string extracomment="Selections">Selections</string>
         </property>
         <property name="whatsThis">
          <string extracomment="Selections">Selections</string>
         </property>
         <property name="text">
          <string>Selections</string>
         </property>
         <property name="icon">
          <iconset resource="PorosityAnalyser.qrc">
           <normaloff>:/images/treeView.png</normaloff>:/images/treeView.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSelections">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string extracomment="Selections">Selections</string>
         </property>
         <property name="statusTip">
          <string extracomment="Selections">Selections</string>
         </property>
         <property name="whatsThis">
          <string extracomment="Selections">Selections</string>
         </property>
         <property name="text">
          <string>Selections</string>
         </property>
         <property name="icon">
          <iconset resource="PorosityAnalyser.qrc">
           <normaloff>:/images/selection_list.png</normaloff>:/images/selection_list.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>858</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="PorosityAnalyser.qrc"/>
 </resources>
 <connections/>
</ui>
